testing/hyperparams/test_get_set_hyperparams.py

```diff
@@ -142,9 +175,40 @@ def test_meta_step_mixin_should_set_hyperparams():
     assert p.wrapped.get_hyperparams()['somestep_hyperparam'] == SOME_STEP_HP_VALUE
 
 
-def test_meta_step_mixin_should_set_hyperparams_space():
+def test_meta_step_mixin_update_hyperparams_should_update_meta_step_hyperparams():
     p = SomeMetaStepMixin(SomeStep())
+    p.set_hyperparams(HyperparameterSamples({
+        META_STEP_HP: META_STEP_HP_VALUE,
+        SOME_STEP_HP: SOME_STEP_HP_VALUE
+    }))
+
+    p.update_hyperparams(HyperparameterSamples({
+        META_STEP_HP: META_STEP_HP_VALUE + 1
+    }))
 
+    assert isinstance(p.hyperparams, HyperparameterSamples)
```

Comment:
Use this for the BaseStep docstring, but without CONSTANTS, rather with "strings" just to simplify the docstring example for demonstration. And the assert isn't "isinstance" in the docstring, but rather, checking the value of it.

Reply:
I have already changed the docstring code example of the base step. it is similar to this unit test. removed the useless isinstance. I am already checking the value of it. resolving